applications/plugins/SofaDistanceGrid/src/SofaDistanceGrid/components/forcefield/DistanceGridForceField.inl

```diff
@@ -67,6 +67,12 @@ void DistanceGridForceField<DataTypes>::init()
                                     nx.getValue(),ny.getValue(),nz.getValue(),
                                     box.getValue()[0],box.getValue()[1]);
 
+    if (grid == nullptr)
+    {
+        sofa::core::objectmodel::BaseObject::d_componentState.setValue(sofa::core::objectmodel::ComponentState::Invalid);
+        return;
```

Comment:
maybe adding a msg_Error() to warn user the Distance grid has not been created and why.

Reply:
It is a relevant suggestion. I added a generic message, and the reason is already the load function: https://github.com/sofa-framework/sofa/blob/6dbc39db7f7ba999783dee6d281fa2f76334ee7c/applications/plugins/SofaDistanceGrid/src/SofaDistanceGrid/DistanceGrid.cpp#L121